src/index.html

```diff
@@ -1,14 +1,40 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+    rel="stylesheet">
+  </head>
+  <body>
+    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header_logo">
+        <img src="images/logo.png" alt="">
+      </div>
+      <nav class="header_nav">
+        <ul class="header_nav_ul">
+          <li><a href="#" class="is-active">APPLE</a></li>
```

Comment:
Починай новий тег з нового рядка для кращої читабельності
```suggestion
          <li>
            <a href="#" class="is-active">
              APPLE
            </a>
         </li>
```